R/ORFs_helpers.R

```diff
@@ -351,8 +351,9 @@ orfID <- function(grl, with.tx = FALSE) {
 }
 
 
-#' Get the unique set of orfs
+#' Get the unique set of groups in a GRangesList
 #'
+#' Normally used for ORFs and leaders.
```

Comment:
Sometimes \code{\link{GRangesList}} groups might be identical, for example ORFs from different isoforms can have identical ranges. Use this function to reduce these groups to unique elements in \code{\link{GRangesList}} \code{grl}, without names and metacolumns.